aries-site/src/pages/templates/filtering.mdx

```diff
@@ -1,124 +1,122 @@
 
-## When to use
+## Examples
 
 ### Filters in a layer
 
-When applying many filters at once, it is helpful to present them in a consolidated [Layer](/components/layer).
+When applying many filters at once, it is helpful to present them in a consolidated layer.
 
 <Example
   code={[
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js',
     'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringLists/mockData.js',
   ]}
-  showResponsiveControls={['laptop', 'mobile']}
+  docs="https://v2.grommet.io/data"
+  showResponsiveControls={['fullScreen']}
   template
 >
   <FilteringLists />
 </Example>
 
-### Filtering with selectable results
+### Exposed filter in toolbar
 
-The summary displayed beneath the search input should always reflect the user's actions. When filtering is combined with selection, the summary text should adjust for the following scenarios:
-
-- No filters, no selections &mdash; The summary should read "X items", where X is the total number of items.
-- Filters applied, no selections &mdash; The summary should read "X results of Y items".
-- Selections made &mdash; The summary should read "X of Y items selected", where Y is the total number of results that meet the filter criteria or the total number of items if no filtering has been applied.
+If it's expected that a specific filter will be most frequently used, it can be useful to expose it in the toolbar.
 
 <Example
-  code={[
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringTable/FilteringTable.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringTable/FilterServers.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringTable/mockData.js',
-  ]}
-  showResponsiveControls={['laptop', 'mobile']}
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/QuickFilterToolbar.js"
+  docs="https://v2.grommet.io/data"
   template
+  height="auto"
+  showResponsiveControls={['fullScreen']}
 >
-  <FilteringTable />
+  <QuickFilterToolbar />
 </Example>
 
-## Guidelines
+### Selectable results
 
-Filtering should help users quickly narrow down data sets by selecting certain parameters to define the relevant data bounds. Different types of inputs may better suited for filtering depending on the type of data.
+The summary displayed beneath the toolbar should always reflect the user's actions. When filtering is combined with selection, the summary text should adjust for the following scenarios.
```

Comment:
Im wondering if we should move this up with the rest of the guidance then link the example.

Reply:
That's fair, moved it up.